OctoKit/Milestone.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+@objc public enum MilestoneState: Int {
+    case Open
+    case Closed
+}
+
+@objc public class Milestone: NSObject {
+    public var url: NSURL?
+    public var htmlURL: NSURL?
+    public var labelsURL: NSURL?
+    public var id: Int
+    public var number: Int?
+    public var state: MilestoneState?
```

Comment:
This should use the same state as `Issue` and also map the same way. `IssueState` is a `String` enum and this is an `Int` enum. The should both be `String` enums.